semantic_conventions/logs/media.yaml

```
@@ -0,0 +1,49 @@
+groups:
+  - id: log
+    prefix: log
```

```suggestion
  - id: attributes.log.stream
    prefix: log.stream
```

Do you see more attributes being added under "log" only? Right now, the `I/O Stream` table is built using `log`. If later someone add new attributes here that section will be updated automatically. Is that intended?

Maybe make it specific to I/O stream attributes, to avoid such potential issue?

---

I see the point here however wouldn't that change the current field to `log.stream.iostream` (from `log.iostream`) 🤔 ?

---

Yes, that is the point basically. But to avoid stalling this, let's then keep as is and maybe before stabilizing this can be looked again.

---

Let's just then change the `id` to `attributes.log`

---

Sure! We can also consider it at https://github.com/open-telemetry/semantic-conventions/issues/134.